core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -936,20 +938,20 @@ class CivilizationInfo {
     }
 
     fun addProtectorCiv(otherCiv: CivilizationInfo) {
-        if(!this.isCityState() or !otherCiv.isMajorCiv() or otherCiv.isDefeated()) return
-        if(!knows(otherCiv) or isAtWarWith(otherCiv)) return //Exception
+        if (!isCityState() || !otherCiv.isMajorCiv() || otherCiv.isDefeated()) return
+        if (!knows(otherCiv) || isAtWarWith(otherCiv)) return //Exception
 
         val diplomacy = getDiplomacyManager(otherCiv.civName)
         diplomacy.diplomaticStatus = DiplomaticStatus.Protector
     }
 
     fun removeProtectorCiv(otherCiv: CivilizationInfo) {
-        if(!this.isCityState() or !otherCiv.isMajorCiv() or otherCiv.isDefeated()) return
```

Comment:
Astounding. The [doc](https://kotlinlang.org/docs/basic-types.html#bitwise-operations) says bitwise operators cannot be applied to Boolean, so why did this even compile before?